core/components/Stead.lua

```diff
@@ -22,8 +22,10 @@ Stead.setBoard = function(board)
   Stead.board = board
 end
 
-Stead.new = function(gob_gid, pos_table, parent)
+Stead.new = function(pos_table, parent)
   local self = setmetatable({},Stead)
+  self.parent = parent
+  self.children = {}
```

Comment:
`Stead`'s ambiguous and dubious capability for handling worldspace/screenspace/boardspace/scenegraph positioning all at once was a large hindrance when it was actually used. Much more work is needed here, or a new approach.